indico/modules/events/registration/controllers/management/tickets.py

```diff
@@ -73,6 +75,12 @@ def _process(self):
                 'scope': 'registrants',
             }
         }
+        sig_rvs = values_from_signal(
+            signals.event.registration.generate_regform_ticket_config_qr_code.send(self.regform, qr_data=qr_data),
+            as_list=True,
+        )
+        if len(sig_rvs) == 1:
```

Comment:
Do we really need to let plugins replace the data? I'd rather have them extend it. So basically update the original `qr_data` with the dict(s) returned by the signal handler(s).